src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -167,6 +167,17 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                     ]}
                     showPageSizeOptions={false}
                     {...norwegian}
+                    getTrProps={(state: any, rowInfo: any) => {
```

Comment:
Besides being an illegal function (lambdas in rendering JSX) this is also an untyped function. Why?